Source/WebCore/loader/ResourceLoadInfo.h

```diff
@@ -35,52 +35,53 @@ namespace WebCore::ContentExtensions {
 
 enum class ActionCondition : uint32_t {
     None = 0x00000,
-    IfTopURL = 0x20000,
-    UnlessTopURL = 0x40000,
-    IfFrameURL = 0x60000,
-    UnlessFrameURL = 0x80000
+    IfTopURL = 0x40000,
+    UnlessTopURL = 0x80000,
+    IfFrameURL = 0x140000,
+    UnlessFrameURL = 0x180000
 };
-static constexpr uint32_t ActionConditionMask = 0xE0000;
+static constexpr uint32_t ActionConditionMask = 0x1C0000;
 
 enum class ResourceType : uint32_t {
-    Document = 0x0001,
-    Image = 0x0002,
-    StyleSheet = 0x0004,
-    Script = 0x0008,
-    Font = 0x0010,
-    SVGDocument = 0x0020,
-    Media = 0x0040,
-    Popup = 0x0080,
-    Ping = 0x0100,
-    Fetch = 0x0200,
-    WebSocket = 0x0400,
-    Other = 0x0800,
-    CSPReport = 0x10000,
+    TopDocument = 0x0001,
+    ChildDocument = 0x0002,
+    Image = 0x0004,
+    StyleSheet = 0x0008,
+    Script = 0x0010,
+    Font = 0x0020,
+    SVGDocument = 0x0040,
+    Media = 0x0080,
+    Popup = 0x0100,
+    Ping = 0x0200,
+    Fetch = 0x0400,
+    WebSocket = 0x0800,
+    Other = 0x1000,
+    CSPReport = 0x2000,
```

Comment:
Since we are changing values, Other makes most sense at the end.
```suggestion
    CSPReport = 0x1000,
    Other = 0x2000,
```